### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Vadoomz**

### CODE REVIEW

**Comment by Vadoomz** [comment]
I have a little bit different result as in description. Is it ok?

File: 21.2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+SELECT * FROM users WHERE age > 30 LIMIT 2
```
Comment (klymenok) [inline 21.2.txt:1]: There shouldn't be a limit here

File: 21.5.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+SELECT age, count(*) as count FROM users GROUP BY age HAVING count() >1 ORDER BY count DESC, age ASC
```
Comment (klymenok) [inline 21.5.txt:1]: You can filter here by aggregated value in `count`: `... HAVING count > 1 ...`

**Review by klymenok — APPROVED** [review]
Looks good, but check my comments

**Pull request merged**